services/distribution/src/main/java/app/coronawarn/server/services/distribution/assembly/io/IO.java

```diff
@@ -48,8 +48,7 @@ public static void makeNewFile(File root, String name) {
         throw new IOException("Could not create " + name + ", file already exists");
       }
     } catch (IOException e) {
-      logger.error("Failed to create file: {}", name, e);
-      throw new RuntimeException(e);
+      throw new RuntimeException("Failed to create file: " + name, e);
```

Comment:
We propose the less generic `UncheckedIOException` which extends `RuntimeException` and also wraps an `IOException`

Reply:
Replaced with UncheckedIOException.